net/strongswan/files/ipsec.init

```diff
@@ -178,6 +186,16 @@ config_conn() {
 	[ -n "$local_firewall" ] && ipsec_xappend "  leftfirewall=$local_firewall"
 	[ -n "$remote_firewall" ] && ipsec_xappend "  rightfirewall=$remote_firewall"
 
+	case "$fragmentation" in
+	yes|accept|force|no)
+		ipsec_xappend "  fragmentation=$fragmentation" ;;
+	esac
+
+	case "$closeaction" in
+	none|clear|hold|restart)
+		ipsec_xappend "  closeaction=$closeaction" ;;
```

Comment:
is there any method or benefit in having a warning printed for the *) cases for these two? Are these hand edited uci config files, or written via tooling/LuCI? Just a thought, might not be useful.

Reply:
> is there any method or benefit in having a warning printed for the *) cases for these two? Are these hand edited uci config files, or written via tooling/LuCI? Just a thought, might not be useful.

I hand-edit my own configs... I don't know how common that is.  Was thinking that adding sanity checking and error reporting might be useful at some point, but it doesn't seem to be very common in the UCI scripts so maybe that's a dead-end.

Reply:
Looked some more and indeed there's a function called `warning()` that burps a message to `stderr`, but it's not used very frequently.